pysd/py_backend/vensim/vensim2py.py

```diff
@@ -831,7 +863,7 @@ def translate_vensim(mdl_file):
     #>>> translate_vensim('../../tests/test-models/tests/limits/test_limits.mdl')
 
     """
-    with open(mdl_file, 'rU') as in_file:
+    with open(mdl_file, 'rU', encoding='UTF-8') as in_file:
```

Comment:
Will this break if users import models they've translated with older versions of pysd? I know some people have made modifications between translation steps...